src/optionsMenu/MicTab.tsx

```diff
@@ -132,6 +132,19 @@ export function MicTab() {
         <option value={"stereo-normal"} label="Stereo" />
         <option value={"stereo-flipped"} label="Stereo - Flipped" />
       </select>
+      <div className="form-check">
+        <input
+          className="form-check-input"
+          type="checkbox"
+          checked={state.processing}
+          onChange={(e) => {
+            dispatch(MixerState.setMicProcessingEnabled(e.target.checked));
+          }}
+        />
+        <label className="form-check-label">
+          Apply Mic Processing (Default: On)
```

Comment:
Why the `(Default: On)`? Surely it being on by default shows that?

Also, while I'm here, does this get reset if you turn off Pro Mode:tm:? If not, should it?

Reply:
This isn't (code-wise) a Pro Mode :tm: feature. It doesn't reset at all, aside from if settings get forgotten in local storage.

This was my idea behind the Default On, given it's rather important based on changing use cases/situations, but isn't easily noticed (you can't hear it), I thought i'd put that there to remind people what it's normally set to.

Reply:
I was tempted to add more of the default texts (or maybe very down-the-line make them auto generated)

Reply:
Hrm - should this be a Pro Mode :tm: feature? If you put it on the master mic tab, you could get non-Pros messing with it, not knowing what it does.

Reply:
Hmm, in my mind it's a toss up, because this is the sort of thing that you could possibly want to use without all the pro-mode features (the UI ones at least). It's not like it's a show stopper if they untick it. I'd rather have it in the home of mic settings than on an island on it's own.

It's not in training to touch it, so hopefully people don't.